ucx.spec

```diff
@@ -91,11 +98,17 @@ The acronym UCX stands for "Unified Communication X".
 Requires: %{name}%{?_isa} = %{version}-%{release}
 Summary: Header files required for developing with UCX
 
+%if (0%{?suse_version} > 0)
```

Comment:
@brianjmurrell please advise if this is ok or hacky.  It appears to work for the build.  At present, we've been using the system version (1.11) on leap

Reply:
Hrm.  I'm on the fence about this.  When we build a package for a platform, we really should be trying to build apples-to-apples packages.

In this case however, at runtime UCX would be coming from MLNX, not SUSE, and not the artifacts of this PR.  I'm not sure how UCX is packaged for MOFED on Leap 15 though.  I would not be at all surprised to learn that MLNX are not building apples-to-apples packages for SUSE distros either.

I guess the ultimate question is does building like this on Leap 15 allow the resulting `libuc{s,t,p}-devel` packages to then be used to build mercury and then allow that mercury to run with MOFED UCX on Leap 15?

If we are going to be claiming to support UCX with SUSE and MOFED, the above really needs to be verified.